src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/block/flower/WakebloomBlock.java

```diff
@@ -12,8 +14,14 @@
 import net.minecraft.world.level.material.Material;
 
 public class WakebloomBlock extends FlowerBlock {
```

Comment:
Either add comment referencing MinecraftForge/MinecraftForge#9140 or wait with this PR until it is merged